### PR TITLE
migrate code from googleapis/java-dataproc

### PR DESCRIPTION
**Pull request by bourgeoisor**
- samples: scaffold pom.xml files (#97)
- chore(deps): update dependency com.google.cloud:libraries-bom to v4.3.0 (#109)
- chore(deps): update dependency com.google.cloud.samples:shared-configuration to v1.0.13 (#110)
- chore(deps): update dependency com.google.cloud.samples:shared-configuration to v1.0.14 (#117)
- chore(deps): update dependency com.google.cloud:libraries-bom to v4.4.0 (#119)
- chore(deps): update dependency com.google.cloud.samples:shared-configuration to v1.0.15 (#124)
- chore(deps): update dependency com.google.cloud:libraries-bom to v4.4.1 (#126)
- chore(deps): update dependency com.google.cloud:libraries-bom to v5 (#149)
- chore(deps): update dependency com.google.cloud:libraries-bom to v5.1.0 (#150)
- chore(deps): update dependency com.google.cloud.samples:shared-configuration to v1.0.16 (#157)
- chore(deps): update dependency com.google.cloud.samples:shared-configuration to v1.0.17 (#161)
- chore(deps): update dependency com.google.cloud:libraries-bom to v5.2.0 (#171)
- chore(deps): update dependency com.google.cloud:libraries-bom to v5.3.0 (#175)
- chore(deps): update dependency com.google.cloud:libraries-bom to v5.4.0 (#188)
- chore(deps): update dependency com.google.cloud:libraries-bom to v5.5.0 (#198)
- chore(deps): update dependency com.google.cloud.samples:shared-configuration to v1.0.18 (#216)
- chore(deps): update dependency com.google.cloud:libraries-bom to v5.7.0 (#212)
- chore(deps): update dependency com.google.cloud:libraries-bom to v6 (#227)
- chore(deps): update dependency com.google.cloud:libraries-bom to v7 (#231)
- chore(deps): update dependency com.google.cloud:libraries-bom to v7.0.1 (#237)
- chore(deps): update dependency com.google.cloud:libraries-bom to v8 (#240)
- chore(deps): update dependency com.google.cloud:libraries-bom to v8.1.0 (#249)
- samples: feat: adding CreateCluster sample for Dataproc (#1734)
- samples: fix: small updates to dataproc sample, test and pom (#1738)
- samples: feat: dataproc quickstart and createcluster (#1908)
- samples: feat: add cli functionality to dataproc quickstart (#2047)
- samples: feat: dataproc instantiate inline workflow sample (#2360)
- samples: changed template tag (#2532)
- samples: feat: adding hadoopfs and autoscaling samples (#3262)
- samples(test): fix required dependencies for tests
-  docs: change relative URLs to absolute URLs to fix broken links. (#260)
- chore(deps): update dependency com.google.cloud:libraries-bom to v9 (#266)
- chore(deps): update dependency com.google.cloud:libraries-bom to v9.1.0
- chore(deps): update dependency com.google.cloud:libraries-bom to v10
- adding submit job sample and updating submit job in quickstart (#284)
- removed extra old code (#285)
- chore(deps): update dependency com.google.cloud:libraries-bom to v11
- chore(deps): update dependency com.google.cloud.samples:shared-configuration to v1.0.21 (#306)
- chore(deps): update dependency com.google.cloud:libraries-bom to v12 (#310)
- test(deps): update dependency junit:junit to v4.13.1
- chore(deps): update dependency com.google.cloud:libraries-bom to v12.1.0 (#320)
- chore(deps): update dependency com.google.cloud:libraries-bom to v13 (#332)
- chore(deps): update dependency com.google.cloud:libraries-bom to v13.1.0 (#338)
- test(deps): update dependency com.google.truth:truth to v1.1 (#333)
- chore(deps): update dependency com.google.cloud:libraries-bom to v13.2.0 (#344)
- chore(deps): update dependency com.google.cloud:libraries-bom to v13.3.0 (#347)
- chore(deps): update dependency com.google.cloud:libraries-bom to v13.4.0 (#356)
- chore(deps): update dependency com.google.cloud:libraries-bom to v14 (#363)
- chore(deps): update dependency com.google.cloud:libraries-bom to v15 (#366)
- chore(deps): update dependency com.google.cloud:libraries-bom to v15.1.0 (#373)
- chore(deps): update dependency com.google.cloud:libraries-bom to v16 (#382)
- chore(deps): update dependency com.google.cloud:libraries-bom to v16.2.0 (#406)
- chore(deps): update dependency com.google.cloud:libraries-bom to v16.2.1 (#418)
- test(deps): update dependency com.google.truth:truth to v1.1.2 (#424)
- chore(deps): update dependency com.google.cloud:libraries-bom to v16.3.0 (#423)
- chore(deps): update dependency com.google.cloud:libraries-bom to v16.4.0 (#438)
- test(deps): update dependency junit:junit to v4.13.2 (#441)
- chore(deps): update dependency com.google.cloud:libraries-bom to v17 (#452)
- chore(deps): update dependency com.google.cloud:libraries-bom to v18 (#455)
- chore(deps): update dependency com.google.cloud:libraries-bom to v18.1.0 (#467)
- chore(deps): update dependency com.google.cloud:libraries-bom to v19 (#470)
- chore(deps): update dependency com.google.cloud:libraries-bom to v19.1.0 (#482)
- chore(deps): update dependency com.google.cloud:libraries-bom to v19.2.1 (#484)
- fixing machine type for new default image (#490)
- chore(deps): update dependency com.google.cloud.samples:shared-configuration to v1.0.22 (#495)
- chore(deps): update dependency com.google.cloud:libraries-bom to v20 (#506)
- chore: update QuickstartTest to use StdOutCaptureRule (#376)
- chore(deps): update dependency com.google.cloud:libraries-bom to v20.1.0 (#513)
- chore(deps): update dependency com.google.cloud:libraries-bom to v20.2.0 (#532)
- chore(deps): update dependency com.google.cloud:libraries-bom to v20.4.0 (#542)
- chore(deps): update dependency com.google.cloud:libraries-bom to v20.5.0 (#559)
- test(deps): update dependency com.google.truth:truth to v1.1.3 (#561)
- chore(deps): update dependency com.google.cloud:libraries-bom to v20.6.0 (#577)
- chore(deps): update dependency com.google.cloud.samples:shared-configuration to v1.0.23 (#576)
- chore: migrate to owlbot (#582)
- chore(deps): update dependency com.google.cloud:libraries-bom to v20.7.0 (#589)
- chore(deps): update dependency com.google.cloud:libraries-bom to v20.8.0 (#601)
- chore(deps): update dependency com.google.cloud:libraries-bom to v20.9.0 (#613)
- chore(deps): update dependency com.google.cloud:libraries-bom to v21 (#641)
- chore(deps): update dependency com.google.cloud:libraries-bom to v22 (#654)
- chore(deps): update dependency com.google.cloud:libraries-bom to v23 (#669)
- chore(deps): update dependency com.google.cloud:libraries-bom to v23.1.0 (#707)
- chore(deps): update dependency com.google.cloud:libraries-bom to v24 (#728)
- chore(deps): update dependency com.google.cloud.samples:shared-configuration to v1.2.0 (#742)
- chore(deps): update dependency com.google.cloud:libraries-bom to v24.1.0 (#751)
- chore(deps): update dependency com.google.cloud:libraries-bom to v24.1.1 (#752)
- chore(deps): update dependency com.google.cloud:libraries-bom to v24.1.2 (#758)
- chore(deps): update dependency com.google.cloud:libraries-bom to v24.2.0 (#770)
- chore(deps): update dependency com.google.cloud:libraries-bom to v24.3.0 (#788)
- build(deps): update dependency org.sonatype.plugins:nexus-staging-maven-plugin to v1.6.9 (#792)
- build(deps): update dependency org.sonatype.plugins:nexus-staging-maven-plugin to v1.6.10 (#793)
- build(deps): update dependency org.sonatype.plugins:nexus-staging-maven-plugin to v1.6.11 (#794)
- chore(deps): update dependency com.google.cloud:libraries-bom to v24.4.0 (#805)
- chore(deps): update dependency com.google.cloud:libraries-bom to v25 (#815)
- sample: changes in three tests to resolve flaky tests and presubmit errors in samples (#819)
- chore(deps): update dependency com.google.cloud:libraries-bom to v25.1.0 (#825)
- build(deps): update dependency org.sonatype.plugins:nexus-staging-maven-plugin to v1.6.13 (#843)
- chore(deps): update dependency com.google.cloud:libraries-bom to v25.2.0 (#845)
- chore(deps): update dependency com.google.cloud:libraries-bom to v25.3.0 (#852)
- chore(deps): update dependency com.google.cloud:libraries-bom to v25.4.0 (#866)
- chore(deps): update dependency com.google.cloud:libraries-bom to v26 (#883)
- build(deps): update dependency org.apache.maven.plugins:maven-deploy-plugin to v3 (#890)
- chore(deps): update dependency com.google.cloud:libraries-bom to v26.1.0 (#902)
- chore(deps): update dependency com.google.cloud:libraries-bom to v26.1.1 (#905)
- chore(deps): update dependency com.google.cloud:libraries-bom to v26.1.2 (#920)
- chore(deps): update dependency com.google.cloud:libraries-bom to v26.1.3 (#960)

Fixes #issue

> It's a good idea to open an issue first for discussion.

- [ ] I have followed [Sample Format Guide](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/main/SAMPLE_FORMAT.md)
- [ ] `pom.xml` parent set to latest `shared-configuration`
- [ ] Appropriate changes to README are included in PR
- [ ] API's need to be enabled to test (tell us)
- [ ] Environment Variables need to be set (ask us to set them)
- [ ] **Tests** pass:   `mvn clean verify` **required**
- [ ] **Lint**  passes: `mvn -P lint checkstyle:check` **required**
- [ ] **Static Analysis**:  `mvn -P lint clean compile pmd:cpd-check spotbugs:check` **advisory only**
- [ ] Please **merge** this PR for me once it is approved.

### CODE REVIEW

File: dataproc/pom.xml [hunk 1]
```diff
@@ -0,0 +1,54 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
```
Comment (averikitsch) [inline dataproc/pom.xml:2]: Parent pom.xml is not needed

Comment (bourgeoisor) [reply dataproc/pom.xml:2]: Done!

**Comment by bourgeoisor** [comment]
Reverting inclusive language commit. Justification: that's the verbiage used by the `com.google.cloud.dataproc.v1.ClusterConfig` import (e.g. the `setMasterConfig()` method)

**Review by Shabirmean — APPROVED** [review]
LGTM!

**Review by davidcavazos — APPROVED** [review]

**Pull request merged**